### PR TITLE
PAYF-7017docs update

### PR DESCRIPTION
**Pull request by jonhageman**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1658,6 +1680,219 @@ <h4 id="example-get-all-patient-types">Example: Delete Payment Profile Response<
...
+            </pre>
+            <p>&nbsp;</p>
+
+            <h1 id='stripe-elements'>Stripe Elements</h1>
+            <p>Stripe Elements is a set of prebuilt UI components needed to process a payment with this api. Their full documentation is <a href="https://stripe.com/docs/payments/elements" target="_blank">here</a>. 
+                The following is a list of the highlights to use: </p>
+                <ol>
+                    <li>Reference Stripe.js in your code.</li>
+                    <li>Initialize Stripe Elements with the <code>paymentProcessorSecret</code> from the <code>PaymentDto</code> in the CreatePayment response.</li>
+                    <li>Use the <code>stripe.confirmPayment</code> which attaches a payment method to the Stripe payment intent.</li>
```
Comment (MetaSequoia87) [inline index.html:1891]: 'Use the stripe.confirmPayment method, which attaches'

File: index.html [hunk 2]
```diff
@@ -1658,6 +1680,219 @@ <h4 id="example-get-all-patient-types">Example: Delete Payment Profile Response<
...
+                <a href="#key-vault-response">KeyVaultResponse</a>
+            </p>
+
+            <h4>Example Response</h4>
+            <pre class="center-column">
+{
+    "N": "oSavKivsDmHseYWFu8zH............1ygwWf9aFy/kuDLxVIzRrRnyKW3U",
+    "E": "AQAB",
+    "Algorithm": "RS256"
+    "KeySize": "2048"
```
Comment (MetaSequoia87) [inline index.html:1720]: KeySize is Int, not sure this comes across with "" - it's in a JSON object. Could we verify this?

File: index.html [hunk 3]
```diff
@@ -1166,36 +1223,28 @@ <h4 id="example-get-all-patient-types">Example: Get Payments Response</h4>
...
-        creditCardType : '1',
         transactionDate : 2000-01-23T04:56:07.000+00:00,
         merchantName : 'merchantName',
-        transactionType : 'phonepayment',
-        transactionStatusId : 6,
         authorizedAmount : '132.00',
         creditCardBrand : 'visa', 
         id : 1,
-        creditCardLast4 : '1234'
+        creditCardLast4 : '1234',
```
Comment (MetaSequoia87) [inline index.html:1232]: 'id' (line 1231 & 1244) is a Guid, not an int, so maybe we want the example more like the Guid on line 1294.

File: index.html [hunk 4]
```diff
@@ -703,7 +748,7 @@ <h2 id="payment-full-dto">PaymentFullDto</h2>
                 </thead>
                 <tbody>
                     <tr>
-                        <td>id</td>
+                        <td>Id</td>
```
Comment (MetaSequoia87) [inline index.html:751]: while we're at it. we could also caps 'Id' on line 728

Comment (jonhageman) [reply index.html:756]: updated

File: index.html (hunk 2 above)
Comment (jonhageman) [reply index.html:1725]: updated

File: index.html [hunk 5]
```diff
@@ -1658,6 +1680,219 @@ <h4 id="example-get-all-patient-types">Example: Delete Payment Profile Response<
...
+        "Status": "failed",
+        "Id": "286271dd-7970-4a48-8c85-33ddc97cb91e",
+        "OriginalTransactionId": "2093ec57-7ca5-4ce5-a725-ab9ed4369db9"
+    }
+}
+            </pre>
+            <p>&nbsp;</p>
+
+            <h1 id='stripe-elements'>Stripe Elements</h1>
+            <p>Stripe Elements is a set of prebuilt UI components needed to process a payment with this api. Their full documentation is <a href="https://stripe.com/docs/payments/elements" target="_blank">here</a>.
```
[the remark this answers is not in the document]
Comment (jonhageman) [reply index.html:1892]: updated

File: index.html (hunk 3 above)
Comment (jonhageman) [reply index.html:1237]: updated

**Review by MetaSequoia87 — APPROVED** [review]
lgtm

**Pull request merged**